### PR TITLE
Finished the static version

### PR DESCRIPTION
**Pull request by yosefbeder**
I tried to do what's only necessary here.
I hope you like it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (M0ST4FA) [inline index.html:7]: You didn't include the link to the style cheat.

Comment (yosefbeder) [reply index.html:7]: Uh

Comment (yosefbeder) [reply index.html:7]: Thanks I will add it, now

**Comment by M0ST4FA** [comment]
Ok I love it

**Pull request merged**